packages/core/src/discovery.js

```diff
@@ -90,7 +103,7 @@ export function createRequestFinishedHandler(network, {
         });
 
         log.debug(`- sha: ${resource.sha}`, meta);
-        log.debug(`- mimetype: ${resource.mimetype}`, meta);
+        log.debug(`- mimetype: ${resourceMimeType}`, meta);
```

Comment:
Not necessary by this point because we pass `resourceMimeType` into `ceateResource` above, so `resource.mimetype === resourceMimeType`.